release-notes/20220818-show-versions.md

```diff
@@ -0,0 +1,13 @@
+<!-- markdownlint-disable MD041 -->
+The `subctl show` command now lists the versions of all deployed Submariner components, instead of just the custom resources.
+
+Example output:
+
+```bash
+   ✓ Showing versions
+  COMPONENT               REPOSITORY           VERSION
+  submariner-gateway      quay.io/submariner   0.13.1
+  submariner-routeagent   quay.io/submariner   0.13.1
+  submariner-globalnet    quay.io/submariner   0.13.1
+  submariner-operator     quay.io/submariner   0.13.1
+```
```

Comment:
This doesn't seem to jive with https://github.com/submariner-io/subctl/issues/219 - it fixed incorrectly reported versions.

```suggestion
The `subctl show` command now correctly reports component image versions when image overrides were specified in `join`.
```